<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateBranchDialog</class>
 <widget class="QDialog" name="CreateBranchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Branch</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid silver;
    border-radius: 6px;
    margin-top: 6px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 8px;
    padding: 0px 5px 0px 5px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gbName">
     <property name="title">
      <string>Name</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lBranch">
        <property name="text">
         <string>Branch</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leBranch"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbBaseOn">
     <property name="title">
      <string>Base On</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbHead">
        <property name="text">
         <string>HEAD</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbBranch">
        <property name="text">
         <string>Branch</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbBranch"/>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="rbTag">
        <property name="text">
         <string>Tag</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cbTag"/>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="rbCommit">
        <property name="text">
         <string>Commit</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cbCommit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbTrack">
        <property name="text">
         <string>Track</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbForce">
        <property name="text">
         <string>Force</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbSwitchToNew">
        <property name="text">
         <string>Switch to new branch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbDescription">
     <property name="title">
      <string>Description</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>268</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbOk">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbOk</sender>
   <signal>clicked()</signal>
   <receiver>CreateBranchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CreateBranchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
